src/introduction.md

```diff
@@ -184,6 +184,10 @@ For example:
      PsyArXiv.
      [doi:10.31234/osf.io/w6z79](https://doi.org/10.31234/osf.io/w6z79)
 
+#### Eye-tracking
+
+-   (publication forthcoming)
+
```

Comment:
I would refrain from inserting this boilerplate here, and just send a PR whenever the publication has a DOI assigned.